### DISCUSSION TITLE
Welcome to PasswordManagement Discussions!

### DISCUSSION THREAD

**Discussion by Madara9744** [Madara9744/PasswordManagement#3] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Madara9744** [comment, marked as answer]
Suggestions accepted for the Roadmap.

**Reply by rohankumardubey** [reply]
let's gather all the dependencies for persistent low latency storage mechanism which would be the core requirement.

**Reply by Madara9744** [reply]
> let's gather all the dependencies for persistent low latency storage mechanism which would be the core requirement.

yes, lets get in that direction. I see mapDb will be suitable enough.
